terraform/dns.tf

```diff
@@ -405,29 +400,34 @@ locals {
       value    = "2a01:4f9:c012:8178::"
     },
 
-    # Mailserver configuration for `mail-test.nixos.org`
+    # Mailserver configuration for `nixos.org`
+    # TODO: remove the 2 MX records for improvmx below in favor of this once
+    # we're ready to switch to the new mailserver:
+    # https://github.com/NixOS/infra/issues/485
+    # {
+    #   hostname = "nixos.org"
+    #   type     = "MX"
+    #   value    = "umbriel.nixos.org"
+    # },
     {
-      hostname = "mail-test.nixos.org"
-      type     = "MX"
-      value    = "umbriel.nixos.org"
-    },
-    {
-      hostname = "mail-test.nixos.org"
+      hostname = "nixos.org"
       type     = "TXT"
-      value    = "v=spf1 mx ~all"
+      # TODO: simplify to just a `mx` rule once umbriel is our one and only
+      # mailserver:
+      # https://github.com/NixOS/infra/issues/485
+      # value = "v=spf1 mx ~all"
+      value = "v=spf1 mx include:spf.improvmx.com include:_mailcust.gandi.net ip4:37.27.20.162 ip6:2a01:4f9:c011:8fb5::1 ~all"
     },
     {
-      hostname = "mail._domainkey.mail-test.nixos.org"
+      hostname = "mail._domainkey.nixos.org"
       type     = "TXT"
-      # From `/var/dkim/mail-test.nixos.org.mail.txt` on `umbriel`.
+      # >>> From `/var/dkim/nixos.org.mail.txt` on `umbriel`. <<<
```

Comment:
NOTE: This has not happened yet. My plan:

- Wait for a shipit on this PR.
- Deploy the changes in this PR to umbriel.
- Update this DNS record.
- Ask someone with Terraform access to deploy the DNS changes here.
- Merge this PR.

Reply:
We can also deploy the DKIM records ahead of time.

Reply:
Sorry, could you explain in a little more detail? Don't I need to deploy the SNM config changes in this PR to get umbriel to generate a `/var/dkim/nixos.org.mail.txt` *before* I can update our DNS?

Reply:
I thought there was a way to pregenerate. But they don't make it easy in the module.